exercises/sum-of-multiples/instructions.md

```diff
@@ -0,0 +1,50 @@
+# Instructions
```

Comment:
I wonder if we could shorten this slightly? It's quite big at the moment.

Reply:
Yeah, it's super long. This is the only story I could come up with that made sense with the calculation, but I'm not sure how to keep it comprehensible while cutting it down.